src/base/bittorrent/session.cpp

```diff
@@ -358,7 +360,12 @@ void Session::setSessionSettings()
     sessionSettings.active_lsd_limit = -1;
 
     // Outgoing ports
+#if LIBTORRENT_VERSION_NUM < 10100
     sessionSettings.outgoing_ports = std::make_pair(pref->outgoingPortsMin(), pref->outgoingPortsMax());
+#else
+    sessionSettings.outgoing_port = pref->outgoingPortsMin();
```

Comment:
the session_settings structure is also something that's deprecated now, in favor of the more flexible and extensible settings_pack structure.